tmt/steps/__init__.py

```diff
@@ -392,6 +404,136 @@ def go(self):
         self.verbose('order', self.order, 'magenta', level=3)
 
 
+class Reboot(tmt.utils.Common):
+    """ Reboot guest """
+
+    # Which kind of reboot
+    _hard = False
+
+    # Dictionary containing list of requested phases for each enabled step
+    _phases = None
+
+    # True if interactive login enabled
+    _enabled = False
+
+    def __init__(self, step, order):
+        """ Initialize relations, store the reboot order """
+        super().__init__(parent=step, name='reboot')
+        self.order = order
+
+    @classmethod
+    def command(cls, method_class=None, usage=None):
+        """ Create the reboot command """
+        @click.command()
+        @click.pass_context
+        @click.option(
+            '--hard', metavar='REBOOT_HARD', is_flag=True,
+            help='Hard reboot of provisioned machine.')
+        @click.option(
+            '--name', metavar='MACHINE_NAME', multiple=False,
```

Comment:
I am a bit out of the loop with this pull request. Could you please tell me what this option will be used for? What will be the name here? For example container ID?

Reply:
According to:
https://github.com/psss/tmt/pull/794#discussion_r665341082

it should be just identification of machine. It depends on provision step used here.